Documentation/ApiOverview/Routing/Introduction.rst

```diff
@@ -20,28 +16,66 @@ When TYPO3 serves a request, it maps the incoming URL to a specific page or acti
 For example it maps an URL like :samp:`https://example.org/news` to the News page. This process of
 determining the page and/or action to execute for a specific URL is called "Routing".
 
-Additionally, routing will take care of beautifying URL parameters, for example converting
+The input of a Route is made up of several components; some components can also be split further
+into sub-components.
+
+Routing will also take care of beautifying URI parameters, for example converting
 :samp:`https://example.org/profiles?user=magdalena` to :samp:`https://example.org/profiles/magdalena`.
 
+..  _routing-terminology:
 
 Key Terminology
 ===============
 
 
-.. index:: Routing; Route
+..  index:: Routing; Structure
+
+Given a complex link (`URI`, `Uniform Resource Identificator`) like
+
+    `https://subdomain.example.com:80/en/about-us/our-team/john-doe/publications/index.xhtml?utm_campaign=seo#start`
+
+all of its components can be broken down to:
+
++----------+------------+----------+-----+------+----------------------+--------------------+----------------+----------------+-------+------------------+----------------+----------------+------------------------+
+| https:// | subdomain. | example. | com | :80  | /en                  | /about-us/our-team | /john-doe      | /publications/ | index | .xhtml           | ?utm_campaign= | seo            | #start                 |
++==========+============+==========+=====+======+======================+====================+================+================+=======+==================+================+================+========================+
+| Protocol | Subdomain  | Domain   | TLD | Port | Site Language Prefix | Slug               | Enhanced Route                                             |                |                |                        |
```

Comment:
Maybe

    abbr:`TLD (Top-Level Domain)`
    
? This explains the abbreviation (or write it out).

Additionally, why "Site Language Prefix" et al instead of "Site language prefix"?

Reply:
Please make a follow up if desired.